neonvm/controllers/virtualmachine_controller.go

```diff
@@ -694,6 +713,18 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 
 		// set VM phase to running if everything scaled
 		if cpuScaled && ramScaled {
+			// update status by CPUs used in the VM
```

Comment:
when `cpuScaled=true` and  `memScaled=true`, it means that we did not perform any action in `VmScaling` since it was already scaled in the previous reconcile cycle. In other words - there is nothing to do and no need to update any statues other than moving `.status.phase` to `VmRunning`.

Reply:
You're right. However, the fact that we didn't update status use in the scaling phase causes #453. Also see my comment on the issue. What do you think?